src/main/java/org/frawa/elmtest/run/ElmTestRunConfiguration.java

```diff
@@ -0,0 +1,102 @@
+package org.frawa.elmtest.run;
+
+import com.intellij.execution.Executor;
+import com.intellij.execution.configurations.ConfigurationFactory;
+import com.intellij.execution.configurations.RunConfiguration;
+import com.intellij.execution.configurations.RunConfigurationBase;
+import com.intellij.execution.configurations.RunProfileState;
+import com.intellij.execution.runners.ExecutionEnvironment;
+import com.intellij.openapi.options.SettingsEditor;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.InvalidDataException;
+import com.intellij.openapi.util.WriteExternalException;
+import org.jdom.Attribute;
+import org.jdom.Element;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+
+import static org.frawa.elmtest.run.ElmTestConfigurationFactory.RUN_ICON;
+
+public class ElmTestRunConfiguration extends RunConfigurationBase {
+
+    Options options = new Options();
+
+    ElmTestRunConfiguration(Project project, ConfigurationFactory factory, String name) {
+        super(project, factory, name);
+    }
+
+    @NotNull
+    @Override
+    public SettingsEditor<? extends RunConfiguration> getConfigurationEditor() {
+        return new ElmTestSettingsEditor(getProject());
+    }
+
+    @Override
+    public void checkConfiguration() {
+    }
+
+    @Nullable
+    @Override
+    public RunProfileState getState(@NotNull Executor executor, @NotNull ExecutionEnvironment executionEnvironment) {
+        return new ElmTestRunProfileState(executionEnvironment, this);
+    }
+
+    static class Options {
+        String elmFolder;
+        String elmTestBinary;
```

Comment:
`elmTestBinary` can be removed from this class since it is now part of the IntelliJ project-level Elm settings.

Reply:
Cleaned.